.github/workflows/run-ank.yml

```diff
@@ -0,0 +1,30 @@
+name: RunAnk
+
+on: pull_request
+
+jobs:
+  type_checking:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          architecture: x64
+      - name: Validate gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Clean build
```

Comment:
:clap: @i-walker !!! Just a few comments. This step is not necessary because of the following one.

Reply:
Another reason: it can lead to confusion because there isn't a purpose to create an artifact, just checking the doc with Ank.

Reply:
I deleted the wrapper check. We need to build the project, because we're using kapt and HKT.

Reply:
Once Meta is in Arrow we can remove that line for sure :)